static/app/components/replays/replayPlayer.tsx

```diff
@@ -76,10 +82,30 @@ function BasePlayerRoot({
   return (
     <Centered ref={windowEl} data-test-id="replay-window">
       <div ref={viewEl} data-test-id="replay-view" className={className} />
+      {fastForwardSpeed ? (
+        <FastForwardBadge>
+          <Tooltip title={t('Fast forwarding')}>
+            <IconArrow size="sm" direction="right" />
+            {fastForwardSpeed}x
+          </Tooltip>
+        </FastForwardBadge>
+      ) : null}
     </Centered>
   );
 }
 
+const FastForwardBadge = styled('div')`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  background: ${p => p.theme.purple100};
+  opacity: 0.75;
+  color: ${p => p.theme.textColor};
+  padding: ${space(0.5)};
+  border-bottom-left-radius: ${p => p.theme.borderRadius};
+  border-top-right-radius: ${p => p.theme.borderRadius};
```

Comment:
Seems like there was an alignment issue in the arrow. Can we use something like

```css
display: grid;
grid-template-columns: max-content max-content;
gap: ${space(0.5)};
align-items: center;
```

This get's things nice and aligned.

Reply:
yes much better! there were some shenanigans with the <Tooltip> too, but here's the change:

**Before:**
<img width="170" alt="before" src="https://user-images.githubusercontent.com/187460/163629527-028c9aea-cd57-4ee3-85b8-27e64cc02405.png">


**After:**
<img width="156" alt="after" src="https://user-images.githubusercontent.com/187460/163629530-647ce0f7-20b5-492c-806f-72b6fc767569.png">

Reply:
That's great!. Design nit, but could maybe use some padding on the right and left too

Reply:
yeah, got tighter in there with the grid.

roomy now:
<img width="161" alt="Screen Shot 2022-04-15 at 2 50 56 PM" src="https://user-images.githubusercontent.com/187460/163636135-51512cd7-cf9e-4529-bb43-0c869adb3afc.png">

@Jesse-Box hasn't seen this yet. But we'll be back either with a proper icon, or to do some other kind of ui indicator treatment. `@posthog/react-rrweb-player` does an overlay over the whole video for example.